docs/stack.md

```diff
@@ -100,6 +100,33 @@ Other tools such as alembic ([https://alembic.sqlalchemy.org/](https://alembic.s
 
 #### Debouncing Mutations
 
+If we expect a mutation to be dispatched several times in quick succession, and if the last instance of the mutation is the only one required to generate the correct state, then only the final mutation should be sent to the back-end, to reduce the load and network traffic.
+This is the case for instance for mutations that are sent on every key press in an input.
```

Comment:
Only idempotent mutations are candidates for debounce.

Reply:
Some mutations that need to be dispatched in quick succession are a better fit for the batch dispatcher, eg. a counter mutation.

Reply:
Will update the docs for completeness. Regarding the batch dispatcher, should it be opt-in? All mutations are batched (with a 10ms window)